<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTcpEdgeTestDataView_GetHisCommon</class>
 <widget class="QWidget" name="CTcpEdgeTestDataView_GetHisCommon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CTcpEdgeTestDataView_DataSubs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QCheckBox" name="__Devices__">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableWidget" name="_Devices_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>设备实例ID(DeviceId)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备列表(Devices)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../TcpEdgeServer/TcpEdgeServer.qrc">
          <normaloff>:/Resources/add.png</normaloff>:/Resources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../TcpEdgeServer/TcpEdgeServer.qrc">
          <normaloff>:/Resources/delete.png</normaloff>:/Resources/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../../TcpEdgeServer/TcpEdgeServer.qrc"/>
 </resources>
 <connections/>
</ui>
